qiskit/opflow/gradients/circuit_gradients/lin_comb.py

```diff
@@ -178,28 +188,27 @@ def _prepare_operator(
             if not isinstance(operator[-1], StateFn) or operator[-1].is_measurement:
                 raise ValueError("The given operator does not correspond to an expectation value")
             if operator[0].is_measurement:
+                meas = deepcopy(operator.oplist[0])
+                meas = meas.primitive * meas.coeff
```

Comment:
Can we rename to measurement?

Reply:
Tbh I'd prefer to keep `meas`. We also use `meas_op` and `aux_meas_op`. If we wanted to be consistent then we'd have to write out all those variable names.

Reply:
Ok